includes/class-wcpdf-frontend.php

```diff
@@ -116,6 +116,7 @@ public function download_invoice_shortcode( $atts ) {
 		global $wp;
 
 		// Get $order
+		$order = '';
```

Comment:
`$order` is never a string, so setting it to an empty string can lead to type-related issues down the road.

The real problem is in this check:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/c9f467de65f49165007a10c9d5039b4be4e7c017/includes/class-wcpdf-frontend.php#L127

That is also the line that the error was referring to. The code is checking `is_object` before checking if it exists at all. Reversing that will solve the error. Furhermore, `!isset()` is not necessary since `!isset()` is a subset of `empty()`.

Reply:
Is this what you were referring to: 27915ee9d110e874e73c7ea70c39e701cd4bc204?

Reply:
Yes!